test/integration/stats_integration_test.cc

```diff
@@ -314,7 +315,8 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithRealSymbolTable) {
   // 2019/11/01  8859     35221       36000   build: switch to libc++ by default
   // 2019/11/15  9040     35029       35500   build: update protobuf to 3.10.1
   // 2019/11/15  9040     35061       35500   upstream: track whether cluster is local
-  // 2019/12/10  8779     35053       35000   use var-length coding for name lengths
+  // 2019/12/20  8779     35053       35000   use var-length coding for name lengths
```

Comment:
This is still a merge issue. To avoid going back and forth on this again can you fix this in your next change?